examples/00-core_examples/00-basic_dpf_implementation.py

```diff
@@ -0,0 +1,217 @@
+#       'string_field', 'support', 'time_freq_scoping_factory', 'time_freq_support', 'unit_system', 'workflow'
+
+######################################################################################################################
+# # ~~~~~~~ USING DPF ~~~~~~~ # #
+
+# An operator is the main object here. You use them to load, operate on, and output data.
+
+# Each operator contains input and output pins, like those in an integrated circuit in electronics, that
+# submit data to the operator and output the computed result from the operator, with respect to its description.
+# Each pin has a number and a name
+
+# The basic steps for using the operators are:
+# ---------------------------------------------------------------------------------------------------------------------
+# 1) Instance the operator. There are two ways for doing this:
+#
+#     1.1) By instantiating any of the derived classes available in the 'ansys.dpf.core.operators' package.
+#     This package also have its subpackages:
+#             - 'ansys.dpf.core.operators.averaging' package
+#             - 'ansys.dpf.core.operators.compression' namespace
+#             - 'ansys.dpf.core.operators.filter' package
+#             - 'ansys.dpf.core.operators.geo' package
+#             - 'aansys.dpf.core.operators.invariant package' package
+#             - 'ansys.dpf.core.operators.logic' package
+#             - 'ansys.dpf.core.operators.mapping' package
+#             - 'ansys.dpf.core.operators.math' package
+#             - 'ansys.dpf.core.operators.mesh' package
+#             - 'ansys.dpf.core.operators.metadata' package
+#             - 'ansys.dpf.core.operators.min_max' package
+#             - 'ansys.dpf.core.operators.result' package
+#             - 'ansys.dpf.core.operators.scoping' package
+#             - 'ansys.dpf.core.operators.serialization' package
+#             - 'ansys.dpf.core.operators.server' package
+#             - 'ansys.dpf.core.operators.utility' package
+
+#      If you believe you'll be using multiple modules available in the same package, its better to import it all
+#      at the beginning
+        from ansys.dpf.core import operators as ops # Import of the dpf core subpackage 'ansys.dpf.core.operators'
+        my_operator = ops.subpackageAttribute.operatorAttribute() # I use the word attribute for any name following a dot
+#      or
+        from ansys.dpf import core as dpf
+        my_operator = dpf.operators.subpackageAttribute.operatorAttribute()
+#      or
+        my_operator = ansys.dpf.core.operators.subpackageAttribute.operatorAttribute()
+
+#      An example on how to create a displacement operator:
+        from ansys.dpf.core import operators as ops
+        my_displacement_operator = ops.result.displacement()
+
+#     1.2) By instantiating  directly with the Operator submodule using the internal name string that indicates the operator
```

Comment:
So, this for example here is not a best practice. So we can mention it, but say that it should only be used when the Python version of the operator is unavailable, like if they made a custom one (but even then they can generate the Python code for their operator, so they should prefer using that).